### DISCUSSION TITLE
40회 반성문

### DISCUSSION THREAD

**Discussion by mun-kyeong** [BDD-CLUB/challenge-every-week-posting#394] [6. 반성문]
개강하니까 과제 쳐내는 것 만으로도 너무 체력소모가 크네요,,,ㅜㅜㅜㅜㅜㅠㅠㅜ
우선 지금 작성중인 글은 plato calendar 2 회고록 겸 작업 과정입니다!
이번주 키글챌 때까지 작성완료해서 올려보겠습니당..!! 
[플라토 캘린더 2 깨알홍보](https://chromewebstore.google.com/detail/plato-calendar-2/enpgcofknkhbihhookpjdblbfggnahgf)

**Comment by shkisme** [comment]
개강시즌이라 다들 바쁘시군요... 화이팅입니다!!!
플라토 캘린더!!!! 학교 다녔으면 넘나 애용했을 거 같네요 ㅎㅎ 



레포에 별도 찍어놨습니다 🤩

**Reply by mun-kyeong** [reply]
헉 이럴수가,, 감사합니당🥹!!!! 정신이 아직 학교수업 커리큘럼을 따라가기엔 벅차다고 울고잇네요 ㅋㅋ쿠ㅠㅜㅠㅜ
조금만더 ,,, 힘내보겠씁니다!!
